### ISSUE TITLE
Improve sql tasks serialization

### ISSUE THREAD

**Issue by idomic** [ploomber/ploomber#489]
When leveraging the etl example template, I could see that the dag is outputing only the sql task names.
Here's how to recreate: {t: dag[t].product.to_json_serializable() for t in dag._iter()}

Out[1]: 
{'download': {'nb': '/home/jovyan/projects/templates/etl/output/download.ipynb',
  'zipped': '/home/jovyan/projects/templates/etl/output/data.7z',
  'extracted': '/home/jovyan/projects/templates/etl/output/extracted'},
 'convert2csv': {'nb': '/home/jovyan/projects/templates/etl/output/convert2csv.ipynb',
  'data': '/home/jovyan/projects/templates/etl/output/data'},
 'upload-users': 'users',
 'upload-comments': 'comments',
 'upload-posts': 'posts',
 'comments-by-post': 'comments_by_post',
 'upvotes-by-location': 'upvotes_by_location',
 'posts-by-length': 'posts_by_length',
 'comments-dump': '/home/jovyan/projects/templates/etl/output/comments-dump.csv',
 'upvotes-dump': '/home/jovyan/projects/templates/etl/output/upvotes-dump.csv',
 'posts-dump': '/home/jovyan/projects/templates/etl/output/posts-dump.csv',
 'comments-plot': {'nb': '/home/jovyan/projects/templates/etl/output/comments-plot.html'},
 'plot-upvotes': {'nb': '/home/jovyan/projects/templates/etl/output/plot-upvotes.html'},
 'posts-plot': {'nb': '/home/jovyan/projects/templates/etl/output/posts-plot.html'}}

When looking at 'upload-users' single task:
'upload-users': SQLiteRelation(('users', 'table'))

Also, not sure if it doesn't show the schema since it wasn't defined.

TLDR : Need to add the kind type of the class to the serialization as well, and the schema if it's readable

**Comment by edublancas** [comment]
is this blocking the telemetry PR?

**Comment by idomic** [comment]
Not blocking, but once it's ready, it'll require a change in the module
